.github/README.md

```diff
@@ -12,11 +12,9 @@
 > May the best app win! 🌠
 
 <p align="center">
-  <img width="128" align="center" src="../assets/fluenthub.png" />
+  <img alt="FluentHub hero image" src="/assets/header.png" />
 </p>
-<h1 align="center">
-  FluentHub
-</h1>
+
 <p align="center">
   <a style="text-decoration:none" href="https://github.com/0x5bfa/FluentHub/actions/workflows/ci.yml">
     <img src="https://github.com/0x5bfa/FluentHub/actions/workflows/ci.yml/badge.svg" alt="CI Status" /></a>
```

Comment:
This section looks a bit weird now that we added the header image.
@0x5bfa should we copy how it is in Files with the download buttons?

Reply:
I don't think so this comes from MS Store and I made the banner before they made.

Reply:
> I don't think so this comes from MS Store and I made the banner before they made.

No I just meant it looks a bit weird with the Store button being where it is.... nevermind...